cmd/sonobuoy/app/gen_plugin.go

```diff
@@ -55,12 +56,13 @@ func init() {
 		Args:   cobra.ExactArgs(1),
 	}
 
-	GenCommand.PersistentFlags().StringArrayVarP(
+	cmd.PersistentFlags().StringArrayVarP(
 		&genPluginOpts.Paths, "paths", "p", []string{".", "./examples/plugins.d/"},
 		"the paths to search for the plugins in. Defaults to . and ./plugins.d/",
 	)
 	// TODO: Other options?
-	GenCommand.AddCommand(cmd)
+	cmd.AddCommand(cmd)
```

Comment:
You're adding it here to itself? Assuming this was just leftover from tinkering and needs removed.

That causes the help to panic:
```
panic: Command can't be a child of itself

goroutine 1 [running]:
github.com/heptio/sonobuoy/vendor/github.com/spf13/cobra.(*Command).AddCommand(0xc000148500, 0xc0000ede80, 0x1, 0x1)
	/Users/jschnake/go/src/github.com/heptio/sonobuoy/vendor/github.com/spf13/cobra/command.go:999 +0x222
github.com/heptio/sonobuoy/cmd/sonobuoy/app.NewCmdGenPlugin(0x2063be0, 0xc0000d4008, 0x1)
	/Users/jschnake/go/src/github.com/heptio/sonobuoy/cmd/sonobuoy/app/gen_plugin.go:64 +0x1bf
github.com/heptio/sonobuoy/cmd/sonobuoy/app.NewSonobuoyCommand(0x2063be0, 0xc0000d4008, 0xc0000aa058)
	/Users/jschnake/go/src/github.com/heptio/sonobuoy/cmd/sonobuoy/app/root.go:41 +0x2ad
main.main()
	/Users/jschnake/go/src/github.com/heptio/sonobuoy/main.go:27 +0x39
```